Tensile/Utilities/ConditionalImports.py

```diff
@@ -0,0 +1,24 @@
+TENSILE_TERM_COLORS: bool = False
+try:
+    from rich import print as _print
+    TENSILE_TERM_COLORS = True
+except ImportError:
+    _print = print
+
+
+def showwarning(message, category, filename, lineno, file=None, line=None):
+    msg = f"{category.__name__}: {message}"
+    if TENSILE_TERM_COLORS:
+        msg = f"[yellow]{msg}[/yellow]"
+    _print(msg)
```

Comment:
does this belong here?

Reply:
This is a tricky one. While it may belong somewhere more appropriate, I would prefer not to expose the conditional print to the whole program, however, it may be necessary.

Reply:
I found a (somewhat hacky) work around to importing `print` appropriately so the location of showwarning has been restored.